packages/acs-ui-common/src/conditional-compilation-sample/index.tsx

```diff
@@ -204,49 +205,47 @@ export const myExtensibleSelector: MyExtensibleSelector = dummyCreateSelector(
   }
 );
 
-/******************************************************************************
- *
- * Some helpers needed for examples above.
- *
- * No conditional compilation examples below this.
+/**
+ * Adding a new feature conditionally often involves a few operations together:
+ * - Introduce a new @beta type
+ * - Introduce a new internal class method that uses the @beta type
+ * - Call the new @beta class method from @public API, but only in beta flavored build.
+ *
+ * Following the general guidelines stated above, we achieve this as follows:
+ * - Unconditionally add the new type
+ *   - Conditionally export the new type in the public API.
+ * - Conditionally add the new class method that uses the conditionally exposed type.
+ * - In the @public method, extract a simple helper function that detects whether the @beta method should be called.
+ *   The implementation of this function uses conditional compilation.
+ *   - Call the method if needed. Because the method and type are defined unconditionally, no conditional
+ *     compliation is needed here.
```

Comment:
correct compliation to compilation.

Reply:
Ouch, auto-submit was enabled.

I can't spell to safe my life :/